gradio/components.py

```diff
@@ -2086,6 +2086,7 @@ def __init__(
         value: Optional[str | List[str] | Callable] = None,
         *,
         file_count: str = "single",
+        file_type: List[str] = ["file"],
```

Comment:
See my comments on below on the `file_type` parameter of the `UploadButton` component